src/components/safe-apps/AddCustomAppModal.tsx

```diff
@@ -65,7 +66,7 @@ const AddCustomAppModal = ({ open, onClose, onSave, safeAppsList }: Props) => {
     if (!isValidURL(debouncedUrl)) return
 
     return fetchSafeAppFromManifest(debouncedUrl, chainId).catch(() => {
-      setError('appUrl', { type: 'custom', message: "The App doesn't support Safe App functionality" })
+      setError('appUrl', { type: 'custom', message: "The app doesn't support Safe App functionality" })
```

Comment:
Don't we regard Safe Apps as proper nouns?

Reply:
If they are, then line 133 would need adjusting.

Reply:
Safe Apps yes, but here it just says "this app isn't a Safe App"